src/Response/CreatePaymentResponse.php

```diff
@@ -49,10 +47,6 @@ public function __construct(array $rawData)
             throw new InvalidArgumentException('Missing "message" in response');
         }
 
-        if (!$this->isOk()) {
```

Comment:
@andrejro2 what's reason?

Reply:
@renat-magadiev I don't remeber to be honest, It's an old times story:) Reverted this change